applications/sink/kafka-sink/src/main/resources/META-INF/dataflow-configuration-metadata.properties

```diff
@@ -1,2 +1,3 @@
 configuration-properties.classes=org.springframework.cloud.fn.consumer.kafka.KafkaPublisherProperties, \
-  org.springframework.boot.autoconfigure.kafka.KafkaProperties
+  org.springframework.boot.autoconfigure.kafka.KafkaProperties, \
+org.springframework.boot.autoconfigure.kafka.KafkaProperties$Producer
```

Comment:
I wonder if there are other apps that use config props that have nesting and we just did not notice until this one?

Reply:
Created https://github.com/spring-cloud/stream-applications/issues/475